### ISSUE TITLE
feat: Prisma Integration?

### ISSUE THREAD

**Issue by jsteve507** [capacitor-community/sqlite#315]
**General Information**
Hello, I have been using prisma for quite a while for my vue web app. I really enjoy the ease of use and the learning curve is really not that hard. This is the first time, i would like to develop my vue app into mobile platform via Capacitor, especially with this plugin. I am sure, that i asked too many for this. But, i don't have knowledge to build kind of like adapter. I've noticed that this plugin has docs for **TypeORM**, which is super nice! But, i've left it. Because, prisma helps my productivity a lot efficient than it (This is just personal opinion). I only ask, if you could give me something like what should i do first.

**Describe the solution you'd like**
Docs for prisma integration with sample project like typeorm.

**Describe alternatives you've considered**
If this is a big to ask, I kinda need a lead/way for this.

**Additional context**
In short, I would like to integrate Prisma Way for capacitor-community/sqlite.

**Comment by jepiqueau** [comment]
@jsteve507 thanks for looking at the plugin. Would be necessary do develop a provider for capacitor like it has been done for TypeORM.

**Comment by jepiqueau** [comment]
@jsteve507 I will not develop it myself. Someone did it for TypeOrm. feel free to do it yourself or contact prisma to ask them to do it. I have enough work to develop and maintain this plugin
